bundles/org.openhab.binding.zoneminder/README.md

```diff
@@ -1,184 +1,277 @@
-| function        | Text   | Text corresponding the value in ZoneMinder: None, Monitor, Modect, Record, Mocord, Nodect  |
-| capture-daemon  | Switch | Run state of ZMC Daemon                                                                    |
-| analysis-daemon | Switch | Run state of ZMA Daemon                                                                    |
-| frame-daemon    | Switch | Run state of ZMF Daemon                                                                    |
+If ZoneMinder authentication is not used, the User and Password parameters should be empty in the *ZoneMinder Server* thing configuration.
+No other configuration is required.
 
-## Manual configuration
+### Authenticated
 
-### Things configuration
+If ZoneMinder authentication is used, first make sure the ZoneMinder user has the **API Enabled** permission set in the ZoneMinder Users configuration.
+Then, enter the user name and password into the ZoneMinder Server thing configuration.
 
+## Discovery
+
+The server bridge must be added manually.
+Once the server bridge is configured with a valid ZoneMinder host name or IP address, 
+all monitors associated with the ZoneMinder server will be discovered.
+
+## Thing Configuration
+
+### Server Thing
+
+The following configuration parameters are available on the Server thing:
+
+| Parameter | Parameter ID | Required/Optional | Description |
+|-----------|--------------|-------------------|-------------|
+| Host                           | host                        | Required  | Host name or IP address of the ZoneMinder server. |
+| Use secure connection          | useSSL                      | Required  | Use http or https for connection to ZoneMinder. Default is http. |
+| Port Number                    | portNumber                  | Optional  | Port number if not on ZoneMinder default port 80. |
+| Use Default Url Path           | useDefaultUrlPath           | Required  | If ON, use ZoneMinder default of /zm; if OFF, use value from urlPath parameter. |
+| Url Path                       | urlPath                     | Optional  | Use this parameter when not using the ZoneMinder default path of /zm. |
+| Refresh Interval               | refreshInterval             | Required  | Frequency at which monitor status will be updated. |
+| Default Alarm Duration         | defaultAlarmDuration        | Required  | Can be used to set the default alarm duration on discovered monitors. |
+| Default Image Refresh Interval | defaultImageRefreshInterval | Optional  | Can be used to set the image refresh interval on discovered monitors. Leave empty to not set an image refresh interval. |
+| Monitor Discovery Enabled      | discoveryEnabled            | Required  | Enable/disable the automatic discovery of monitors. Default is enabled. |
+| Monitor Discovery Interval     | discoveryInterval           | Required  | Frequency at which the binding will try to discover monitors. |
```

Comment:
The config xml specifies that these are not required.

Reply:
Fixed